tests/ui/treat-err-as-bug/panic-causes-oom-112708.rs

```diff
@@ -1,4 +1,6 @@
-// compile-flags: -Ztreat-err-as-bug
+// This should run with more than one thread, too, but that can cause
+// deadlocks in query stack printing.
+// compile-flags: -Ztreat-err-as-bug -Zthreads=1
```

Comment:
@oli-obk we currently expect to always use `-Zthreads=1` with the parallel compiler

Reply:
oh... hmm... I guess I need to try to run that test in a loop then until it fails...